### PR TITLE
Add getLastLogIdx and getLastLogTerm, wrap LogEntry into AppendEntries

### PR DESCRIPTION
**Pull request by AndrzejRyl**

### CODE REVIEW

File: server/src/main/java/pl/edu/agh/utils/LogArchive.java [hunk 1]
```diff
@@ -41,4 +41,18 @@ public void commitEntry(LogEntry entry) {
         pendingEntries.entrySet().removeIf(e -> e.getKey().getId() == entry.getId());
         commitedEntries.add(entry);
     }
+
+    public int getLasLogIdx() {
+        return pendingEntries.size() + commitedEntries.size();
```
Comment (Hoobie) [inline server/src/main/java/pl/edu/agh/utils/LogArchive.java]: Smart one. I hope that this is what we want :D

Comment (AndrzejRyl) [reply server/src/main/java/pl/edu/agh/utils/LogArchive.java]: Should be ;) I don't remove entries anywhere. I could look at ID of the last one added, but that's simpler ;)

Comment (Hoobie) [reply server/src/main/java/pl/edu/agh/utils/LogArchive.java]: It's ok, I understand it now :D

File: server/src/main/java/pl/edu/agh/utils/LogArchive.java [hunk 2]
```diff
@@ -41,4 +41,18 @@ public void commitEntry(LogEntry entry) {
...
+    }
+
+    public int getLastLogTerm() {
```
Comment (Hoobie) [inline server/src/main/java/pl/edu/agh/utils/LogArchive.java:54]: It seems ok too.

**Review by Hoobie — APPROVED** [review]

**Pull request merged**